blackjack/game.py

```diff
@@ -0,0 +1,127 @@
+from card import Card
+from deck import Deck
+from player import Player
+from dealer import Dealer
+from interface import Interface
+import sys
+
+class Game:
+    """Run the main Blackjack game loop
+
+    Responsibilities:
+    * Starts Game
+    * Starts New Hand if Player has $
+    * Evaluate who wins and loses
+    * Pays out bets
+
+    Collaborators:
+    * Card & Deck
+    * Player & Dealer
+    * Interface for input & output
+    """
+
+    def __init__(self):
+        self.interface = Interface()
+
+    def game_start(self):
+        self.player = Player()
+        self.dealer = Dealer()
+
+
+    def start_hand(self):
+        self.player.bank -= 10
+        self.deck = Deck()
+        self.deck.shuffle_deck()
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
```

Comment:
I get what you're going for here -- this simulates how you actually deal in real life, but we don't necessarily have to do that here. It doesn't change anything statistically. I would change this to let `take_a_hit` specify a number of cards, so you can get 1 or 2 depending on what's going on.